### ISSUE TITLE
Basic Project Setup

### ISSUE THREAD

**Issue by Oliversw** [founders-programme-2/commons#10]
- [x] Create basic react skeleton 
   - We have used [Create-React-App](https://facebook.github.io/create-react-app/) in the past and I would be interested in using it again, since it sets up a lot of really useful things. Really happy to have a discussion about it though!
- [x] Create staging/development branch and set as default branch
- [x] Link to Travis and include badge on readme
- [x] Link to Heroku
   - [ ] Potentially create a staging heroku as well as production? I don't know how to do this however so we can always look at this later.
- [ ] Setup code cov for badge on readme
- [x] Install previously mentioned dependencies #1 

Not sure what else needs to be done before we start writing the actual code, but open to suggestions!

**Comment by Oliversw** [comment]
@ali-7 what was the problem with Code Cov again? I can maybe make a separate issue for that that we can tackle later?

**Comment by ali-7** [comment]
@Oliversw  yes, I didn't continue working on it, open an issue for the Code Cov badge for later maybe as priority 5 or 6. and just keep working now with built-in coverage of react.

**Comment by Oliversw** [comment]
@ali-7 sounds great! I will do that and close this one so it doesn't clutter everything up.
